web/controllers/users_controller.ex

```diff
@@ -0,0 +1,61 @@
+defmodule Notox.UserController do
+  use Notox.Web, :controller
+  alias Notox.User
+
+  def index(conn, _params) do
```

Comment:
Yeah that's dumb. I lost sight of the fact that I'm actually building something. Shameful.